scripts/generate_kernel_ref_data.py

```diff
@@ -1,3 +1,8 @@
+import rascal
+import rascal.lib as lrl
+from rascal.representations import SphericalInvariants
+from rascal.models import Kernel
+from rascal.utils import ostream_redirect
```

Comment:
Auto-reformating issue? This is now above the `sys.path.insert`

Reply:
I know why it happens now... Do you have an idea in how to configure pep8 to avoid such behavior ?

Reply:
no, sorry